cypress/support/authorityApi.js

```diff
@@ -11,9 +11,10 @@ const setAxiosDefaults = () => {
   Axios.defaults.headers.put['Content-Type'] = 'application/json';
 };
 
-export const getAuthorities = async (name, idToken) => {
+export const getAuthorities = async (name, feideId, idToken) => {

```

Comment:
vi hadde en diskusjon om dette før sommeren også, om å implementere samme kode i både frontend og e2e. Hvis endpointet endrer seg må vi oppdatere to steder, og siden det er to implementasjoner kan det skje at frontend feiler mens e2e virker eller motsatt.
Argumentet _for_ var vel at dette bare er oppsett som gjøres _før_ testene?
Jeg syns uansett at det burde være en last resort, og at alt som kan gjøres i appen bør gjøres via appen, feks via cy command. 

**tl;dr: kan gi en kort forklaring hvorfor dette trengs?**

Reply:
Vi trenger å sette opp forutsetningene for testene og være sikker på at de er de samme hver gang vi kjører testene. Bruker vi appen er det ting vi ikke kan gjøre i oppsettet fordi det ikke skal være mulig å gjøre for brukerne. Eksempelvis sette opp app-admin-bruker som nå kun sjekker på unit.no, men som jeg regner med er en full rolle i det nye rollesystemet. 
Jeg ønsker å kunne rydde opp i testdata både før og etter vi har kjørt en test for å være sikker på at det ikke ligger noen tidligere testdata igjen som kan forstyrre testen. Ved å bruke appen blir dette vanskeligere og i noen tilfeller ikke mulig.
I tillegg blir det som å bruke appen for å teste seg selv; hvis appen oppretter ting feil så blir det vanskeligere å oppdage det i testene.

Har tenkt å innvolvere Rurik i denne diskusjonen når han er tilbake fra ferie, så foreslår vi tar et møte i neste uke om det.

Reply:
Forøvrig er jeg ikke uenig i at duplisering av kode er et problem og bør ungås. Hadde en tanke at kanskje vi burde lage et bibliotek for api-kallene slik at vi slipper dupliseringen.

Reply:
Apropos roller: 
Per nå har vi lagt inn 2 admin-roller som allerede brukes i dev (ikke mer sjekk på unit-epost osv):
"Institution-admin" - kan endre sin egen kundeinstitusjon (inkl. legge til andre "Institution-admin"-brukere)
"App-admin" - kan opprette kundeinstitusjoner og legge til "Institution-admin"-brukere for alle kundeinstitusjoner

Reply:
enig i at vi burde tatt et møte på dette!